### DISCUSSION TITLE
Mistral JSON mode support?

### DISCUSSION THREAD

**Discussion by akulmehta** [prism-php/prism#108] [Q&A]
Mistral supports JSON mode as described in their documentation at https://docs.mistral.ai/capabilities/json_mode/. Can we use this in Prism currently in some manner?

**Comment by sixlive** [comment]
Yes! I'll create an issue for it!

**Reply by akulmehta** [reply]
Any way I can help with this? I was trying to figure out which part of the codebase the parameter could be passed into but couldn't - sorry.

**Reply by sixlive** [reply]
I'm in the middle of an internal refactor, specifically for structured output. I should be done by the end of the weekend. Then I should be able to either point you in the direction or just knock it out after or during the refactor.
